oving all files into correct repo but git history is in old repo

tatsqui opened the pull request:
# Personal Portfolio Site
Congratulations! You're submitting your assignment!

## Comprehension Questions
Question | Answer
:------------- | :-------------
Did you have to resolve any issues when running the HTML Validator? If so, what were they? | Yes. The HTML navigator does not like my navigation links but I am not sure what I need to change to make it like them. 
Why is it important to consider and use semantic HTML? | For folks who use technology that depend on having a semantically navigable structure. It also helps other engineers because it makes your code more readable. 
How did you decide to structure your CSS? | I decided to structure it in the same order as what is on my page and I used some comments to refer to what the purpose was. 
What was the most challenging piece of this assignment? | Making a modern looking design from scratch, and figuring out breakpoints and how to have dynamic website without JS. I feel like my website has a long way to go still. 
Describe one area that you gained more clarity on when completing this assignment | I gained more understanding of how to plan a layout from the outside in. 
**Optional** |
Did you deploy to GitHub Pages? If so, what is the URL to your website? |

tatsqui committed:
oving all files into correct repo but git history is in old repo

tatsqui commented:
Oh also, I made all of my commits in a different repo because for some odd reason I didn't think this was going to be an assignment we submitted? I think I saw it on the calendar last week and I just made a local repo on my computer so here is a link so that you can see that I did indeed make more than one commit! https://github.com/tatsqui/portfolio-website

dHelmgren commented:
# Personal Portfolio Site
## What We're Looking For

Feature | Feedback
:------------- | :-------------
**Baseline** |
Appropriate Git Usage | Yes
Answered comprehension questions | yes
Page fully loads | yes
No broken links (regular or images) | yes
Includes at least 4 pages and styling | yes
**HTML** |
Uses the high-level tags for organization: `header, footer, main` | yes
Appropriately using semantic tags: `section, article`, etc. | yes
All images include alternate text | yes
**CSS** |
Using class and ID names in style declarations | yes
Style declarations are DRY | yes
**Overall** | Good work overall! It's clear to me that the learning goals for this assignment were met.



For this project, the instructors are giving a spot-check review, looking for big patterns rather than going through your code line-by-line. If you’d like a more in-depth review, please reach out to me!

tatsqui committed:
note to self

<!–– https://www.w3schools.com/w3css/tryit.asp?filename=tryw3css_templates_parallax&stacked=h -->
